plugins/builds/index.js

```diff
@@ -753,56 +872,81 @@ exports.register = (server, options, next) => {
                     return createInternalBuild(internalBuildConfig);
                 }
 
-                return createExternalBuild(externalBuildConfig);
+                return createExternalBuild(externalBuildConfig).then(async (result) => {
+                    const externalJobId = workflowGraph.nodes
+                        .find(n => n.name === nextJobName).id;
+                    const externalBuildId = result.builds.find(b => b.jobId === externalJobId).id;
+                    const externalBuild = await buildFactory.get({ id: externalBuildId });
+                    const masterParentBuilds = {};
+
+                    masterParentBuilds[result.pipelineId] = {
+                        eventId: result.id,
+                        jobs: { [externalJobName]: externalBuildId }
+                    };
+                    const nextJobs1 = workflowParser.getNextJobs(workflowGraph,
```

Comment:
can we have a meaningful name than `nextJobs` ? eg: `nextJobsToTrigger` 


similar feedback for `nextJobName1` & `result1`